inst/rmd/gimapQCTemplate.Rmd

```diff
@@ -81,17 +82,35 @@ qc_constructs_countzero_bar(gimap_dataset, filter_zerocount_target_col= filter_z
 If this filter is applied, this is the number of pgRNAs that would be filtered out
 
 ```{r}
-qc_filter_zerocounts(gimap_dataset, filter_zerocount_target_col = filter_zerocount_target_col)$reportdf
+potentialFilter1 <- qc_filter_zerocounts(gimap_dataset, filter_zerocount_target_col = filter_zerocount_target_col)
+
+potentialFilter1$reportdf
 ```
 
 ### Filter pgRNAs where there is a low log2 CPM value for the plasmid sample/time point
 
 If this filter is applied, this is the number of pgRNAs that would be filtered out
 
 ```{r}
-qc_filter_plasmid(gimap_dataset)$plasmid_filter_report
+potentialFilter2 <- qc_filter_plasmid(gimap_dataset, filter_plasmid_target_col = filter_plasmid_target_col)
+
+potentialFilter2$reportdf
 ```
 
+### If both filters are applied
+
+```{r}
+combined_filters <- reduce(list(potentialFilter1$filter, potentialFilter2$filter), cbind)
```

Comment:
This is part of description (g) "but then combined the potential filters themselves later on"